src/java.base/share/classes/javax/crypto/JceSecurityManager.java

```diff
@@ -45,9 +47,8 @@
  *
  * @since 1.4
  */
-
 @SuppressWarnings("removal")
```

Comment:
You should remove this annotation now that the dependency on SecurityManager has been removed.

Reply:
Unfortunately, we are still calling AccessController, thus the annotation needs to remain.

Reply:
But if you follow my suggestion you can simply apply it to this line:

```
@SuppressWarnings("removal")
final List<StackFrame> stack = AccessController.doPrivileged(pa).walk(Stream::toList);
```

Reply:
For the static initializer that needs updating:  I could move the code out of the initializer up to the declaration, or I could create a dummy declaration and then assign to INSTANCE.

Reply:
The latter is probably better so you don't have to move the code; @wangweij used this technique quite a bit for JEP 411 refactoring, see https://github.com/openjdk/jdk/commit/508cec7535cd0ad015d566389bc9e5f53ce4103b

Reply:
@seanjmullan , that's what I ended up doing.  I'll have a new revision out as soon as the mach5 finishes.

Reply:
In this case, I name the variable `tmp`. There are cases where the action is a `PrivlegedAction<Void>` and you still have to create a variable for the return value, and then I call it `dummy`.